db/schema.rb

```diff
@@ -60,12 +60,11 @@
   end
 
   create_table "resource_monitors", force: :cascade do |t|
-    t.boolean  "enabled",       default: true
-    t.string   "service"
-    t.datetime "created_at",                   null: false
-    t.datetime "updated_at",                   null: false
-    t.string   "resource_type"
-    t.integer  "resource_id"
+    t.boolean  "enabled",    default: true
```

Comment:
Sorry, one more thing! Could we get this not nullable as well?

Reply:
What benefit would the system have by making this nullable?

Reply:
Without `null: false`, `boolean` fields can be true, false or null. It would be better that they can only be true or false.

Reply:
OIC 👍  made the change